src/components/ProgressBar/ProgressBar.story.js

```diff
@@ -33,4 +34,28 @@ storiesOf(`${GROUPS.COMPONENTS}|ProgressBar`, module)
         </ProgressBar>
       </div>
     ))
+  )
+  .add(
+    'ProgressBar with knobs',
+    withInfo()(() => {
+      const size = select(
+        'Size',
+        {
+          kilo: ProgressBar.KILO,
+          mega: ProgressBar.MEGA,
+          giga: ProgressBar.GIGA
+        },
+        ProgressBar.KILO
+      );
+      const max = number('Maximum number', 10);
```

Comment:
I'd call this "Maximum value" so that the relationship to the "Value" is clearer.